tasks/run-sample-analysis.js

```diff
@@ -39,7 +39,7 @@ module.exports = function(grunt) {
       if (configFile) {
          getConfig = Q.ninvoke(fs, 'readFile', configFile)
             .then(function(configData) {
-               return yaml.safeLoad(configData);
+               return yaml.load(configData);
```

Comment:
In case anyone is wondering why this changed from `safeLoad` to `load`: the `load` method is now "safe" in `js-yaml` 4.0+ according to https://github.com/nodeca/js-yaml/blob/master/CHANGELOG.md#changed-1 / https://github.com/nodeca/js-yaml/blob/master/migrate_v3_to_v4.md#safeload-safeloadall-safedump--load-loadall-dump